WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -58,18 +58,14 @@ import Combine
         }
     }
 
-    lazy var syncContext: NSManagedObjectContext = {
-        return ContextManager.sharedInstance().newDerivedContext()
-    }()
-
-    lazy var service: ReaderPostService = {
-        return ReaderPostService(managedObjectContext: syncContext)
-    }()
+    private var coreDataStack: CoreDataStack {
+        ContextManager.shared
+    }
 
     /// An alias for the apps's main context – temporarily replaces  `newMainContextChildContext` until we have `NSPersistentContainer` support
```

Comment:
While we're at it:

```suggestion
    /// An alias for the apps's main context – temporarily replaces `newMainContextChildContext` until we have `NSPersistentContainer` support
```

Reply:
Actually... After re-reading that comment I wonder what `newMainContextChildContext` is about 🤔 I don't see any other mentions of it in the codebase.

Reply:
> I wonder what newMainContextChildContext is about 🤔

I thought this might be something I didn't know about Core Data (there's a lot I don't know about Core Data 😅 ) so I googled it. The first and only result is a https://github.com/wordpress-mobile/WordPress-iOS/issues/15829

<img width="997" alt="image" src="https://user-images.githubusercontent.com/1218433/199306482-b66e1b4f-1f21-4a3d-8833-7a0505726903.png">.

With that extra reading and more context, I'd say we can remove the comment altogether. After all, we now have `NSPersistentContainer` support.

```suggestion
```

Reply:
Good catch. I'll remove the inaccurate part, but keep the "An alias for the apps's main context" part which is still correct.